app/mailers/patient_mailer.rb

```diff
@@ -195,29 +141,22 @@ def assessment_voice(patient)
     # If the dependets are in a different jurisdiction they may end up with too many or too few symptoms in their response
     contents += I18n.t('assessments.phone.daily3', locale: lang) + patient.jurisdiction.hierarchical_condition_bool_symptoms_string(lang) + '?'
     contents += I18n.t('assessments.phone.daily4', locale: lang)
-    account_sid = ENV['TWILLIO_API_ACCOUNT']
-    auth_token = ENV['TWILLIO_API_KEY']
-    from = ENV['TWILLIO_SENDING_NUMBER']
-    client = Twilio::REST::Client.new(account_sid, auth_token)
+
     threshold_hash = patient.jurisdiction.jurisdiction_path_threshold_hash
     # The medium parameter will either be SMS or VOICE
     params = { prompt: contents, patient_submission_token: patient.submission_token,
                threshold_hash: threshold_hash, medium: 'VOICE', language: lang.to_s.split('-').first.upcase,
                intro: I18n.t('assessments.phone.intro', locale: lang),
                try_again: I18n.t('assessments.phone.try-again', locale: lang),
                thanks: I18n.t('assessments.phone.thanks', locale: lang) }
-    client.studio.v1.flows(ENV['TWILLIO_STUDIO_FLOW']).executions.create(
-      from: from,
-      to: Phonelib.parse(patient.primary_telephone, 'US').full_e164,
-      parameters: params
-    )
-    add_success_history(patient, patient)
+
+    if TwilioSender.start_studio_flow(patient, params)
+      add_success_history(patient, patient)
+    else
+      add_fail_history_sms(patient)
```

Comment:
This looks like it should be `add_fail_history_voice(patient)`. However one of my other comments recommending moving that all into one method, but either way this will need to be fixed to whatever the final method ends up being named.